crypto/argon2/core.h

```diff
@@ -0,0 +1,225 @@
+/*
+ * Argon2 reference source code package - reference C implementations
+ *
```

Comment:
Presumably this has changes to fit into openssl.  The copyright should look like other contribute-code copyrights.